ZenPacks/zenoss/Microsoft/Windows/patches/platform.py

```diff
@@ -114,3 +115,30 @@ def _editDetails(self, info, data):
         info.post_update()
 
     return result
+
+
+@monkeypatch('Products.Zuul.facades.devicefacade.DeviceFacade')
+def getTemplates(self, id):
+    object = self._getObject(id)
```

Comment:
This monkeypatch will affect all devices. Is it intentional or may be a filter by "Windows" only will be better here?